src/Sulu/Bundle/MediaBundle/Media/Manager/DefaultMediaManager.php

```diff
@@ -150,7 +150,7 @@ public function __construct(
         MediaRepositoryInterface $mediaRepository,
         CollectionRepositoryInterface $collectionRepository,
         UserRepositoryInterface $userRepository,
-        ObjectManager $em,
+        EntityManager $em,
```

Comment:
Is there a reason to use the `EntityManager`? If there's not I would prefer `ObjectManager´, since it would be easier to switch to a ODM one day.